src/components/FallbackCandidateServiceTile.jsx

```diff
@@ -13,7 +13,7 @@ const FallbackCandidateServiceTile = ({ slug }) => {
   const cozyURL = new URL(client.getStackClient().uri)
   const app = 'store'
   const nativePath = `/discover/${slug}`
-  const { cozySubdomainType: subDomainType } = client.getInstanceOptions()
+  const { subdomain: subDomainType } = client.getInstanceOptions()
```

Comment:
This concept is still named cozySubdomainType on my local project. Did this changed recently?

Reply:
it depends on how you declare your "injection"... 

If you use : 

If you use: 
```
cozy-data={}
```

then you'll have `subdomain` (https://github.com/cozy/cozy-stack/blob/dd05900e12cce9430faecf869f21e9abaaa702fd/web/apps/serve.go#L342-L374) 

but if you use the : `cozy-data-x={y}` syntax, the app can do what she wants. The cozy-stack will look for a few `var`, then replace the value. So if you write `data-cozy-subdomain-type="{{.SubDomain}}"` then you all have access to the subdomain by reading cozySubdomainType. (cf  https://github.com/cozy/cozy-pass-web/blob/master/src/index.html#L16)